priam/src/main/java/com/netflix/priam/identity/token/DeadTokenRetriever.java

```diff
@@ -67,22 +63,6 @@ public DeadTokenRetriever(
 
         List<String> crossAccountAsgInstances = membership.getCrossAccountRacMembership();
 
-        if (logger.isInfoEnabled()) {

```

Comment:
Nit: Good idea to isolate the changes in future. Easy to cherry pick, revert etc.,